src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -410,6 +410,7 @@ const submitEncryptModeForm: ControllerHandler<
       currency: paymentConfig.defaultCurrency,
       payment_method_types: ['card', 'grabpay', 'paynow'],
       description: form.payments.description,
+      receipt_email: req.body.paymentReceiptEmail,
```

Comment:
should we save this email in our DB? e.g. `form.payments.receipt_email`

can't recall what the conclusion of our discussion (about the scope of the MVP, and also future plans) was wrt this but could be good as a method to identify the individual and help them check their payment status

Reply:
Oh, just remembered, we were talking about warning respondents when they have already made payment on the same form before. Since we don't have the notion of respondent accounts yet, this will mean we'll have to save the emails somewhere to check!

Reply:
sounds good! maybe we should do it in this PR then? since it's clear we want the email to be saved to the DB.

Reply:
As discussed during sync (15 March 2023), we will save the email as plaintext in db for now (implemented in [43c7d92](https://github.com/opengovsg/FormSG/pull/5917/commits/43c7d9288c9dcaf09424c0064cd340153d09987e)) to prevent blocking ourselves. Hashing / encryption can be implemented in the future if needed. Checking with policy team on whether there are any concerns for this.

Reply:
@shawn-ogp from policy team mentioned that emails (which are sensitive normal) should be fine as long as they are encrypted at rest. Since our mongodb is encrypted at rest by default, it should be fine! 👍